[0;33m--- Warning: GDB cannot open serial ports accessed as COMx[0m
[0;33m--- Using \\.\COM9 instead...[0m
[0;33m--- esp-idf-monitor 1.5.0 on \\.\COM9 115200[0m
[0;33m--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H[0m
[0;33m--- 0x4004c0dc: uart_tx_one_char_uart.part.0 in ROM
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpyley7xun"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpyley7xun': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmps5kzfr8j"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmps5kzfr8j': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmprh2c92ci"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmprh2c92ci': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp2l1x_awd"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp2l1x_awd': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpowjviq66"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpowjviq66': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp4wi7fs3j"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp4wi7fs3j': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp5za97td3"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp5za97td3': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpmjcujyq6"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpmjcujyq6': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpbq5wz_j8"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpbq5wz_j8': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- To exit from IDF monitor please use "Ctrl+]". Alternatively, you can use Ctrl+T Ctrl+X to exit.[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- To exit from IDF monitor please use "Ctrl+]". Alternatively, you can use Ctrl+T Ctrl+X to exit.[0m
[0m
Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.3.1\tools/idf_monitor.py", line 11, in <module>
    sys.exit(subprocess.run([sys.executable, '-m', 'esp_idf_monitor'] + sys.argv[1:]).returncode)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "subprocess.py", line 550, in run
  File "subprocess.py", line 1199, in communicate
  File "subprocess.py", line 1262, in wait
  File "subprocess.py", line 1543, in _wait
KeyboardInterrupt
